cylc/flow/time_parser.py

```diff
@@ -163,7 +165,7 @@ def parse_timepoint(self, expr, context_point=None):
 
         expression should be a string such as 20010205T00Z, or a
         truncated/abbreviated format string such as T06 or -P2Y.
-        context_point should be an isodatetime.data.TimePoint object
+        context_point should be an metomi.isodatetime.data.TimePoint object
```

Comment:
nit-pick, not really important I believe... `be a m`

Reply:
yes.